### PR TITLE
pushing to consolidate work

### PR DESCRIPTION
**Pull request by BeniBarra**

### CODE REVIEW

**Review by SorviusN — APPROVED** [review]

File: AzureSkies/AzureSkies/Controllers/FlightsController.cs [hunk 1]
```diff
@@ -38,49 +38,55 @@ public  void  GetFlightInfo(NewSMSFlightDTO incoming)
...
         // PUT: api/Flights/5
-        [HttpPut("{id}")]
-        public async Task<IActionResult> PutFlightInfo(int id, FlightInfo flightInfo)
-        {
-            if (id != flightInfo.Id)
-            {
-                return BadRequest();
-            }
-            _context.Entry(flightInfo).State = EntityState.Modified;
+        //[HttpPut("{id}")]
```
Comment (ghost) [inline AzureSkies/AzureSkies/Controllers/FlightsController.cs:41]: Great job on refactoring this block on code. It is much more readable!

File: AzureSkies/AzureSkies/Controllers/FlightsController.cs [hunk 2]
```diff
@@ -38,49 +38,55 @@ public  void  GetFlightInfo(NewSMSFlightDTO incoming)
...
-                if (!FlightInfoExists(id))
-                {
-                    return NotFound();
-                }
-                else
-                {
-                    throw;
-                }
-            }
+        //    try
```
Comment (ghost) [inline AzureSkies/AzureSkies/Controllers/FlightsController.cs:50]: for readability purposes delete any used code

File: AzureSkies/AzureSkies/Controllers/FlightsController.cs [hunk 3]
```diff
@@ -97,9 +103,9 @@ public async Task<IActionResult> DeleteFlightInfo(int id)
             return NoContent();
         }
 
-        private bool FlightInfoExists(int id)
-        {
-            return _context.FlightInfo.Any(e => e.Id == id);
-        }
+        //private bool FlightInfoExists(int id)
```
Comment (ghost) [inline AzureSkies/AzureSkies/Controllers/FlightsController.cs:106]: please remove any unused code

File: AzureSkies/AzureSkies/DTO/FlightDTO.cs [hunk 4]
```diff
@@ -8,12 +8,12 @@ namespace AzureSkies.DTO
     public class FlightDTO
     {
         // Flight {FlightNumber}
-        public string FlightNumber { get; set; }
+        public string flightDate { get; set; }
         // with {Airline}
-        public string Airline { get; set; }
+        //public string Airline { get; set; }
```
Comment (ghost) [inline AzureSkies/AzureSkies/DTO/FlightDTO.cs:13]: please remove unused code

File: AzureSkies/AzureSkies/Models/Departure.cs [hunk 5]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AzureSkies.Models
+{
+    public class Departure
+    {
+        public int id { get; set; }
```
Comment (ghost) [inline AzureSkies/AzureSkies/Models/Departure.cs:10]: best practice would be to capitalize

File: AzureSkies/AzureSkies/Services/FlightStatusService.cs [hunk 6]
```diff
@@ -13,25 +13,23 @@ public class FlightStatusService : IFlightStatus
...
 
-        public async Task<FlightDTO> AddFlight(string flightNumber, string airline, string date)
+        public async Task<FlightInfo> AddFlight(string flightNumber, string airline, string date)
         {
-            string access_key = "00d536921176e9ab78f646a12b9c15e8";
-            string path = $"http://api.aviationstack.com/v1/flights?{access_key}&" +
-                            $"airlineName={airline}&flightNumber={flightNumber}" +
-                            $"&flightDate={date}";
+            //string access_key = "00d536921176e9ab78f646a12b9c15e8";
+            string path = $"http://api.aviationstack.com/v1/flights?access_key=00d536921176e9ab78f646a12b9c15e8&airlineName={airline}&flightNumber={flightNumber}&flightDate={date}";
```
Comment (ghost) [inline AzureSkies/AzureSkies/Services/FlightStatusService.cs:19]: The access key should not be visible to the internet

Comment (ghost) [reply AzureSkies/AzureSkies/Services/FlightStatusService.cs:19]: put it in a .env file or private field

File: AzureSkies/AzureSkies/obj/Debug/net5.0/Razor/Pages/Error.cshtml.g.cs [hunk 7]
```diff
@@ -1,4 +1,4 @@
-#pragma checksum "C:\Users\Jona\Source\Repos\TheAzurites\AzureSkies\AzureSkies\AzureSkies\Pages\Error.cshtml" "{ff1816ec-aa5e-4d10-87f7-6f4963833460}" "1f02565c188bc0cf60de1bc120acef71a3608055"
+#pragma checksum "C:\Users\locoq\repo\AzureSkies\AzureSkies\AzureSkies\Pages\Error.cshtml" "{ff1816ec-aa5e-4d10-87f7-6f4963833460}" "1f02565c188bc0cf60de1bc120acef71a3608055"
```
Comment (ghost) [inline AzureSkies/AzureSkies/obj/Debug/net5.0/Razor/Pages/Error.cshtml.g.cs:1]: Do these files need to be in a .gitignore?

File: AzureSkies/AzureSkies/Controllers/FlightsController.cs [hunk 8]
```diff
@@ -38,49 +38,55 @@ public  void  GetFlightInfo(NewSMSFlightDTO incoming)
...
             {
-                FlightDate = flightDTO.Date,
-                FlightNumber = flightDTO.FlightNumber,
-                Airline = flightDTO.Airline
+                flightDate = flightInfo.data.flight_date
             };
             return flightInfo;
         }
 
+        [HttpGet("helloworld")]
```
Comment (ghost) [inline AzureSkies/AzureSkies/Controllers/FlightsController.cs:84]: Is this hello world necessary in the finally product?

File: AzureSkies/AzureSkies/Models/Flight.cs [hunk 9]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AzureSkies.Models
+{
+    public class Flight
+    {
+        public int id { get; set; }
```
Comment (ghost) [inline AzureSkies/AzureSkies/Models/Flight.cs:10]: Capitalize

File: AzureSkies/AzureSkies/Models/FlightInfo.cs [hunk 10]
```diff
@@ -8,16 +8,9 @@ namespace AzureSkies.Models
 {
     public class FlightInfo
     {
+        public int id { get; set; }
```
Comment (ghost) [inline AzureSkies/AzureSkies/Models/FlightInfo.cs:11]: Capitilze

**Review by ghost — COMMENTED** [review]
Reviewed by Miriam, Q, Ben (Dave) Arno

**Pull request merged**